.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -140,6 +136,15 @@ jobs:
               ${{ env.AWS_DEFAULT_REGION }} \
               ${{ env.SAMPLE_APP_NAMESPACE }}
           
+              # If the workflow provides a specific ADOT image to test, patch the deployment and restart CW agent related pods
+              if [ ${{ inputs.appsignals-adot-image-name }} != "" ]; then
+                kubectl patch deploy -namazon-cloudwatch amazon-cloudwatch-observability-controller-manager --type='json' \
+                -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/args/0", "value": "--auto-instrumentation-java-image=${{ inputs.appsignals-adot-image-name }}"}]'
```

Comment:
Can you provide link to the template code in CWAgent for this value that is being patched? Is it always guaranteed to be this path?

Reply:
Yes, the image for ADOT and for the Operator are set in this file:
https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/helm/templates/operator-deployment.yaml